runtime/dart_isolate.cc

```diff
@@ -689,7 +692,8 @@ Dart_Isolate DartIsolate::DartIsolateGroupCreateCallback(
               parent_group_data.GetIsolateShutdownCallback())));
 
   TaskRunners null_task_runners(advisory_script_uri,
-                                /* platform= */ nullptr, /* raster= */ nullptr,
+                                /* platform= */ nullptr,
+                                /* raster= */ nullptr,
```

Comment:
The C++ style is to not have the spaces between the comments: https://google.github.io/styleguide/cppguide.html#Function_Argument_Comments

I'm not sure if the linter is smart enough to handle the spaces.

Reply:
I hope it is, because the formatter is adding them =)